PokemonRampUp/Model/Pokemon.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Pokemon.swift
+//  PokemonRampUp
+//
+//  Created by Thushen Mohanarajah on 2023-02-10.
+//
+
+import Foundation
+import UIKit
+
+class Pokemon {
+    let name: String
+    let type: String
+    let hiddenAbilites: [String]
+    let image: UIImage
+    let moves: [String]
+    
+    //
+    // MARK: - Initialization
+    //
+    init(name: String, type: String, hiddenAbilites: [String], image: UIImage, moves: [String]) {
```

Comment:
can you move each property to a new line like this:

init(name: String,
       type: String)